content/en/news/releases/1.7.x/announcing-1.7.2/index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Announcing Istio 1.7.2
+linktitle: 1.7.2
+subtitle: Patch Release
+description: Istio 1.7.2 patch release.
+publishdate: 2020-09-18
+release: 1.7.2
+aliases:
+    - /news/announcing-1.7.2
+---
+
+This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.7.1 and Istio 1.7.2
+
+{{< relnote >}}
+
+## Changes
+
+- **Fixed** a bug that locality load balancer settings were applied inbound clusters unnecessarily. ([Issue #27293](https://github.com/istio/istio/issues/27293))
+
+- **Fixed** unbounded cardinality of Istio metrics for cron job workload. ([Issue #24058](https://github.com/istio/istio/issues/24058))
+
+- **Fixed** Setting the `ISTIO_META_REQUESTED_NETWORK_VIEW` environment variable for a proxy will filter out endpoints that aren’t part of the comma-separated list of networks. This should be set to the local-network on the ingress-gateway used for cross-network traffic to prevent odd loadbalancing behavior. ([Issue #26293](https://github.com/istio/istio/issues/26293))
```

Comment:
load balancing is two words. Separate here and remove the incorrect word from .spelling

Reply:
Not sure if this should be **Fixed** setting (lowercase the s), or if this is something new that's **Added**.

Reply:
Thanks! I have pinged @stevenctl to take a look at the second one.

Reply:
I think it can be considered a fix, but I guess it's more something that enables a fix. I'm fine with *Added*.

Reply:
Confirmed with @stevenctl and let's keep `Fixed` setting here.